[4.1.0] Update policy feature docs and images (#5846)
wso2/docs-apim: en/docs/design/api-policies/attach-policy.md

# Attaching Policies

Follow the instructions below to attach one or more default policies that are shipped with WSO2 API Manager to an API operation(s) of an existing API.

1. Sign in to the WSO2 API Publisher.

    `https://<hostname>:9443/publisher`

2. Click on the API for which you want to attach policies to (e.g., `PizzaShackAPI 1.0.0`). Navigate to **API Configurations** --> **Policies**.

> 3. Under the `API Gateway` section, make sure to select the correct gateway. API Manager supports the following two gateway types: `Regular Gateway (Synapse Gateway)` and `Choreo Connect`. Pick the gateway depending on the gateway that your API is deployed in.

Reviewer: ```suggestion
3. Under the `API Gateway` section, make sure to select the correct Gateway. API Manager supports the following two Gateway types: `Regular Gateway (Synapse Gateway)` and `Choreo Connect`. Select the Gateway depending on the Gateway that your API is deployed in.
```

    [![Select Gateway]({{base_path}}/assets/img/design/api-policies/attach-policies-1.png){: style="width:90%"}]({{base_path}}/assets/img/design/api-policies/attach-policies-1.png)

    !!! note

>         Keep in mind that the policies attached under a given `API Gateway` will be removed when you switch between gateways. But this policy removal will be reflected in your API only if you click on the **Save** button found at the bottom of the Policies page.

Reviewer: ```suggestion
        Keep in mind that the policies attached under a given `API Gateway` will be removed when you switch between Gateways. However, this policy removal will be reflected in your API only if you click on the **Save** button found at the bottom of the Policies page.
```

    !!! tip

>         * Policy list is dependant on the selected gateway (i.e. policy list under `Regular Gateway` is diferent to that of `Choreo Connect`).

Reviewer: ```suggestion
        * The policy list is dependant on the selected Gateway (i.e. policy list under `Regular Gateway` is different to that of `Choreo Connect`).
```

    </table>

    !!! note 

>         You can optionally use the **Apply to all** resources option to attach the same policy to all the resources on Save. This will attach the same policy to all the API operations along with the values you entered to configure the policy (if any). It is important to note that if the policy was applied to the `Request Flow`, it will only be applied to all the `Request Flows` of all operations.

Reviewer: ```suggestion
        You can optionally use the **Apply to all** resources option to attach the same policy to all the resources when you save your settings. This will attach the same policy to all the API operations along with the values you entered to configure the policy (if any). It is important to note that if the policy was applied to the `Request Flow`, it will only be applied to all the `Request Flows` of all operations.
```
